src/isittacoday.ps1

```diff
@@ -1,4 +1,4 @@
-if ([DateTime]::Today.DayOfWeek -eq 'Wednesday')
+if ([DateTime]::Today.DayOfWeek -eq [System.DayOfWeek]::Wednesday)
```

Comment:
good catch

Reply:
agree, great contribution for non-english locales. how should we address the actual messages? currently they are in fluent spanglish

Reply:
While full localization is obviously the long-term goal, my focus here is on ensuring that we at least don't break in non-English locales, even if we aren't in the right language.

I've opened #6 to track.